javascript-local-storage/data.js

```diff
@@ -0,0 +1,14 @@
+/* exported todos */
+
+var todos = [];
+
+var previousTodosJSON = localStorage.getItem('javascript-local-storage');
+if (previousTodosJSON !== null) {
+  // eslint-disable-next-line no-redeclare
+  var todos = JSON.parse(previousTodosJSON);
```

Comment:
you don't want to redeclare the variable, you just need to assign the return value to `todos`